espanol/ritual/ritual.md

```diff
@@ -1,2 +1,3 @@
 Empezó con el ritual de bienvenida para el ejército.
-Dicho ritual incluía los cánticos milenarios de los ganadores de las mil batallas del gran ejército de la República.
\ No newline at end of file
+Dicho ritual incluía los cánticos milenarios de los ganadores de las mil batallas del gran ejército de la República.
+No obstante, aquella república cada vez era gorbernada más oligarquía. No le cabía la menor duda de que en algún momento un pequeño grupo se haría con todo el poder. Eso si no era una sola persona.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.